Bao/documentation page

bnguyen1212 opened the pull request:
## Description
This pull request includes the implementation of the Docs/ Privacy Policy page with basic information on user data privacy and accessibility standards as per non-functional requirement 3.2.3.1. In addition, `CareerBridge_v4.png` logo is proposed with no background, larger size, and no spinning animation. 

## Changes

### `client/index.html`
- changed favicon to `CareerBridge_v4.png`

### `client/public/CareerBridge_v4.png`
- new proposed logo with background removed

### `client/src/App.css`
- changed .logo css:
  - height: 40px &rarr; 70px
  - commented out animation
- miscellaneous auto-formatting

### `client/src/App.tsx`
- added route to `./pages/PrivacyPolicy.tsx` (referenced in `client/src/pages/Docs.tsx`)

### `client/src/components/UI/Navbar.tsx`
- changed NavBar.Brand img src to `/CareerBridge_v4.png`
- miscellaneous auto-formatting

### `client/src/pages/Help.tsx` and `client/src/pages/Contact.tsx`
- fix to automatically scroll to top when component is mounted

### `client/src/pages/Docs.tsx`
- basic information on purpose, getting started, accessibility, and privacy policy added

### `client/src/pages/PrivacyPolicy.tsx`
- new page created with basic privacy policy information under GDPR

## Screenshots
![image](https://github.com/user-attachments/assets/90e2652a-ab15-41cb-ab5d-17dafdaf75ca)

![image](https://github.com/user-attachments/assets/d52b4cc5-28d2-4eef-82c7-18e11d774f93)

## Additional Notes

Please review the changes and provide feedback. Thank you!

han-nwin commented on client/src/components/UI/Navbar.tsx:
Hey Bao can you remove semicolons, they're not needed and distracting

han-nwin commented on client/src/pages/Contact.tsx:
for the scroll to top behavior can you bring it to a separate component and import it. SInce we can reuse that behavior in other pages

han-nwin reviewed (changes requested):
1. No semicolons please
2. Build a separte ScrollToTop.tsx component and import since we can reuse it in other pages

bnguyen1212 commented:
I have made the requested changes:
- Removed unnecessary semicolons
- Moved the `ScrollToTop` behavior to a reusable component in `components/UI/ScrollToTop.tsx`.
- Updated `Docs.tsx`, `PrivacyPolicy.tsx`, `Help.tsx`, and `Contact.tsx` to use the new `ScrollToTop` component.

Please review the changes. Thank you!

han-nwin reviewed (commented):
good job
